src/Manager.php

```diff
@@ -148,4 +145,21 @@ private function isHandled(EntitySearchHandlerInterface $handler, string $class)
         // handles this class.
         return \count($intersect) > 0;
     }
+
+    /**
+     * Get index name.
+     *
+     * @param \LoyaltyCorp\Search\Interfaces\EntitySearchHandlerInterface $handler
+     * @param object $object
+     *
+     * @return string
+     */
+    private function getIndexName(EntitySearchHandlerInterface $handler, object $object): string
```

Comment:
This method kinda kills the seperation we've achieved with the TransformerInterface.

Could we add a method to that interface instead that receives the $object, index name and return the index name that should be used?

Reply:
Ahh! forgot to move when refactoring.